### ISSUE TITLE
Clarify status of Quick Reference Guide, "normative document", and "list of terms" document.

### ISSUE THREAD

**Issue by baskaufs** [tdwg/dwc#265]
This has been moved from https://github.com/tdwg/dwc/issues/256 . 

- [x] The name of the "Normative Document" (as referenced in the DwC repo landing page) needs to be changed to something else. Not everything in it is normative and the document that will actually tell people what is normative is the list of terms document. Maybe "complete term history table"?
- [x] The Quick Reference Guide also needs to have a header section that tells people that it is not part of the standard
- [x]  The Quick Reference Guide needs to have a link to the list of terms document for people who care about the details.  This is blocked by https://github.com/tdwg/dwc/issues/254 (Review draft DwC "list of terms" document).
- [x] When the "list of terms" document exists, it needs to be added to the list of DwC standards documents on the DwC human-readable landing page.
- [x] When the "list of terms" document exists, it needs to be added to the parts of DwC and the list of documents in the rs.tdwg.org repo.

**Comment by tucotuco** [comment]
> * The Quick Reference Guide also needs to have a header section that tells people that it is not part of the standard

I think we have to be very careful how this is worded so as not to confuse people. Rather than say it is not part of the standard, I would spin it in the positive and say that this document is a human readable document derived from the standard (reference) to facilitate the correct usage of Darwin Core terms - or something like that.

**Comment by baskaufs** [comment]
Hmm. One of the issues that keeps coming up repeatedly over they years is that people complain that they don't know what actually is and is not part of the TDWG standards. When we were talking about the QRG in Ottawa, the feeling was that by keeping the QRG document outside of the standard, we were more free to change it and structure it as we wanted without being restricted by any rules imposed by the Vocabulary Maintenance Specification.

I think there may be two options:
1. make the QRG document officially part of the standard. But that would require it to have particular formatting (headers, etc.) and I think that was what we were trying to avoid.
2. keep it out of the standard and say something like "This document is intended to be an easy-to-read summary of the full list of terms maintained as part of the standard." (with a link to the full list of terms document). 

I personally would like some kind of statement saying that the QRG is not one of the documents included in the standard, but I think that if there is a link to the full list of terms and if the DwC standard landing page explicitly lists the documents that are included in the standard, that is probably OK.

**Comment by tucotuco** [comment]
I agree and opt for option 2 with your observations taken into account.

**Comment by tucotuco** [comment]
> * The name of the "Normative Document" (as referenced in the DwC repo landing page) needs to be changed to something else. Not everything in it is normative and the document that will actually tell people what is normative is the list of terms document. Maybe "complete term history table"?

This sounds good to me. I addressed explicitly in the QRG that it is not part of the standard in e3594cb8d785928678771cace64d412faadeafff

**Comment by baskaufs** [comment]
@MattBlissett I have updated the rs.tdwg.org repository to include the new Darwin Core "list of terms" document and also the 2020-08-12 Darwin Core term revisions. Can you please run the script to reload the data into the BaseX server so that dereferencing will start working for these new items?

**Comment by baskaufs** [comment]
Ticked the last box
